controller/notify.go

```diff
@@ -1,18 +1,28 @@
 package controller
 
 import (
+	"context"
 	"fmt"
 
 	"github.com/fabric8-services/fabric8-notification/app"
 	"github.com/fabric8-services/fabric8-notification/collector"
 	"github.com/fabric8-services/fabric8-notification/email"
 	"github.com/fabric8-services/fabric8-notification/jsonapi"
 	"github.com/fabric8-services/fabric8-notification/template"
+	"github.com/fabric8-services/fabric8-notification/token"
 	"github.com/fabric8-services/fabric8-wit/errors"
 	"github.com/fabric8-services/fabric8-wit/log"
 	"github.com/goadesign/goa"
 )
 
+const (
+	CVENotification string = "analytics.notify.cve"
```

Comment:
I guess we should define new Type called NotificatoinType and add constants for all notification_type defined at https://github.com/fabric8-services/fabric8-notification/blob/master/main.go#L106-L113.  Introducing notification_type will trigger more re-factoring in this https://github.com/fabric8-services/fabric8-notification/blob/master/collector/receiver.go#L33 and few more places.
Shall we do this ??  If yes, I would like to do it as separate PR.